module/Application/view/application/index/index.phtml

```diff
@@ -4,7 +4,7 @@
     <p>
         Congratulations! You have successfully installed the
         <a href="https://github.com/zendframework/ZendSkeletonApplication" target="_blank">ZF Skeleton Application</a>.
-        You are currently running Zend Framework version <?= \Application\Module::VERSION ?>.
+        You are currently running Zend Framework version <?php echo \Application\Module::VERSION ?>.
```

Comment:
Well, then, I guess I better fix those. :smile: (I've been trying to convert those whenever I see them.)